dspace-api/src/main/java/org/dspace/content/service/CollectionService.java

```diff
@@ -461,12 +462,21 @@ public int countCollectionsWithSubmit(String q, Context context, Community commu
      *       and does not query the database directly.
      *       This means that results may be stale or outdated until
      *       https://github.com/DSpace/DSpace/issues/2853 is resolved."
-     * 
+     *
      * @param context          DSpace Context
      * @param entityType       limit the returned collection to those related to given entity type
      * @return                 list of collections found
      * @throws SearchServiceException    if search error
      */
     public List<Collection> findAllCollectionsByEntityType(Context context, String entityType)
         throws SearchServiceException;
+
+    /**
+     * Returns total collection archived items
+     *
+     * @param collection       Collection
+     * @return                 total collection archived items
+     * @throws ItemCountException
+     */
+    int countArchivedItem(Collection collection) throws ItemCountException;
```

Comment:
Same here. This method should probably be plural `countArchivedItems` instead of `countArchivedItem`.

Reply:
Done.